0a.md

```diff
@@ -0,0 +1,80 @@
+NIP-0a
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines several new event kinds representing different types of audio track. These events
+are _parameterized replaceable_ as defined in [NIP-01](./01.md) and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+The following kinds are defined by this NIP:
+
+- `31337` represents a music track
+- `31338` represents a podcast episode
+
+The following tags are required:
+
+- `d` is a unique identifier randomly generated by the client.
```

Comment:
So the publishing client will randomly generate the `d` tag? Can the GUID in the optional `i` tag be used here as the `d` tag value?

Reply:
Yes, it should be random. Meaningful d-tags are an antipattern, since you can't really anticipate when something might change, even guids.